docs/topics/Gateway.md

```diff
@@ -504,6 +504,11 @@ Events are payloads sent over the socket to a client that correspond to events i
 | [Guild Role Create](#DOCS_TOPICS_GATEWAY/guild-role-create)                         | guild role was created                                                                                                           |
 | [Guild Role Update](#DOCS_TOPICS_GATEWAY/guild-role-update)                         | guild role was updated                                                                                                           |
 | [Guild Role Delete](#DOCS_TOPICS_GATEWAY/guild-role-delete)                         | guild role was deleted                                                                                                           |
+| [Guild Scheduled Event Create](#DOCS_TOPICS_GATEWAY/guild-scheduled-event-create)    | guild scheduled event was created                                                                                               |
+| [Guild Scheduled Event Update](#DOCS_TOPICS_GATEWAY/guild-scheduled-event-update)    | guild scheduled event was updated                                                                                               |
+| [Guild Scheduled Event Delete](#DOCS_TOPICS_GATEWAY/guild-scheduled-event-delete)    | guild scheduled event was deleted                                                                                               |
+| [Guild Scheduled Event User Create](#DOCS_TOPICS_GATEWAY/guild-scheduled-event-user-create) | user subscribed to a guild scheduled event                                                                               |
+| [Guild Scheduled Event User Delete](#DOCS_TOPICS_GATEWAY/guild-scheduled-event-user-delete) | user unsubscribed from a guild scheduled event                                                                           |
```

Comment:
```suggestion
```
`GUILD_SCHEDULED_EVENT_USER_CREATE` and `GUILD_SCHEDULED_EVENT_USER_DELETE` will be deprecated in the near future so we can remove these

Reply:
We should replace it I guess.